packages/core/core/src/RequestTracker.js

```diff
@@ -1080,34 +1082,72 @@ export default class RequestTracker {
   }
 
   async writeToCache() {
-    let cacheKey = getCacheKey(this.options);
-    let requestGraphKey = hashString(`${cacheKey}:requestGraph`);
-    let snapshotKey = hashString(`${cacheKey}:snapshot`);
+    const cacheKey = getCacheKey(this.options);
+    const requestGraphKey = `requestGraph:${hashString(cacheKey)}`;
```

Comment:
I changed these as it's easier to debug what's happening if the cache folder has useful names